demisto_sdk/commands/common/tests/dependencies_test.py

```diff
@@ -285,7 +286,7 @@ def run_random_methods(repo, current_pack, current_methods_pool, number_of_metho
     all_dependencies = set()
 
     for i in range(number_of_methods_to_choose):
-        chosen_method_index = random.sample(range(1, len(current_methods_pool)), 1)[0]
+        chosen_method_index = random.sample(range(len(current_methods_pool)), 1)[0]
         chosen_method = current_methods_pool[chosen_method_index]
         current_methods_pool.remove(chosen_method)
```

Comment:
since you are choosing just one and you don't care about the index, you can use `random.choice`